usd/src/usd_parser/USDPhysics.hh

```diff
@@ -15,8 +15,8 @@
  *
 */
 
-#ifndef USD_PARSER_PHYSYCS_HH
-#define USD_PARSER_PHYSYCS_HH
+#ifndef SDF_USD_USD_PARSER_USDPHYSYCS_HH_
+#define SDF_USD_USD_PARSER_USDPHYSYCS_HH_
```

Comment:
nit: There is a typo here, `PHYSYCS` should be `PHYSICS` (note the `I` instead of `Y`). Also, according to https://github.com/ignitionrobotics/sdformat/pull/817#discussion_r791031195, I believe the include guard should be `USD_PARSER_USDPHYSICS_HH_`.